stripecardscan-example/README.md

```diff
@@ -9,6 +9,15 @@
 
 <img width="215" height="108" src="https://raw.githubusercontent.com/stripe/stripe-android/master/stripecardscan-example/images/run_project.png" />
 
+### Set up permissions to use the flow
+The stripecardscan module is currently in private beta, and merchants must be manually included in the beta for this demo to work.
+
+If you're from outside Stripe, contact Stripe support to request to be added to the card image verification beta.
+
+If you're internal to Stripe:
```

Comment:
we probably don’t want to expose internal links, probably just say "contact CardScan team for help"?